### PR TITLE
feat(demo): Add minikube cluster support

### PR DESCRIPTION
**Pull request by Techassi**
This PR adds minikube support.

### CODE REVIEW

File: bins/stackablectl/src/cmds/demo.rs [hunk 1]
```diff
@@ -51,6 +52,10 @@ pub struct DemoListArgs {
 pub struct DemoDescribeArgs {
     /// Demo to describe
     #[arg(name = "DEMO")]
+    #[arg(long_help = "Demo to describe
```
[the remark this answers is not in the document]
Comment (Techassi) [reply bins/stackablectl/src/cmds/demo.rs:55]: Yes. For longer help texts I usually opt to use a second `#[arg()]`, but in this case we can combine them.

File: bins/stackablectl/src/cmds/demo.rs [hunk 2]
```diff
@@ -264,5 +290,5 @@ async fn install_cmd(
 }
 
 fn uninstall_cmd(_args: &DemoUninstallArgs, _list: DemoList) -> Result<String, DemoCmdError> {
-    todo!()
+    Ok("Currently not implemented".into())
```
[the remark this answers is not in the document]
Comment (Techassi) [reply bins/stackablectl/src/cmds/demo.rs:293]: Strictly speaking yes. However, I wanted to make it look nicer for end users, as `todo!()` panics. But I can revert this.

File: src/cluster/kind/mod.rs [hunk 3]
```diff
@@ -83,27 +83,24 @@ impl KindCluster {
             .args(["--name", self.name.as_str()])
             .args(["--config", "-"])
             .stdin(Stdio::piped())
-            .stdout(Stdio::null())
-            .stderr(Stdio::null())
+            // .stdout(Stdio::null())
+            // .stderr(Stdio::null())
```
[the remark this answers is not in the document]
Comment (Techassi) [reply src/cluster/kind/mod.rs:87]: Yeah I want to provide a unified progress and error output in the future. For now we can either opt to display this or to keep hiding it.

File: bins/stackablectl/src/cmds/demo.rs [hunk 4]
```diff
@@ -61,6 +66,10 @@ pub struct DemoDescribeArgs {
 pub struct DemoInstallArgs {
     /// Demo to install
     #[arg(name = "DEMO")]
+    #[arg(long_help = "Demo to install
```
[the remark this answers is not in the document]
Comment (Techassi) [reply bins/stackablectl/src/cmds/demo.rs]: Changed in bdcd16c

File: src/cluster/minikube/mod.rs [hunk 5]
```diff
@@ -33,18 +42,28 @@ impl MinikubeCluster {
...
+        // Check if Docker is running
+        check_if_docker_is_running().await.context(DockerSnafu)?;
+
         // Create local cluster via minikube
+        debug!("Creating minikube cluster");
         let minikube_cmd = Command::new("minikube")
             .arg("start")
+            .args(["--driver", "docker"])
             .args(["--nodes", self.node_count.to_string().as_str()])
             .args(["--namespace", self.namespace.as_str()])
```
[the remark this answers is not in the document]
Comment (Techassi) [reply src/cluster/minikube/mod.rs]: Yeah minikube was just setting the default namespace to `--namespace`. I removed that functionality in bdcd16c

File: src/cluster/kind/mod.rs (hunk 3 above)
[the remark this answers is not in the document]
Comment (nightkr) [reply src/cluster/kind/mod.rs:80]: I'd say keep displaying it. Unifying is nice in theory, but it's not exactly like either tool has a stable output format either.

[the remark this answers is not in the document]
Comment (Techassi) [reply src/cluster/kind/mod.rs]: 9539349

File: bins/stackablectl/src/cmds/demo.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (nightkr) [reply bins/stackablectl/src/cmds/demo.rs:297]: The "nice output for end users" would be to do the right thing, or to not expose it at all :P

[the remark this answers is not in the document]
Comment (Techassi) [reply bins/stackablectl/src/cmds/demo.rs:297]: Alright, fair enough :)

[the remark this answers is not in the document]
Comment (Techassi) [reply bins/stackablectl/src/cmds/demo.rs]: Commented out in 5e1f224

**Pull request merged**